config/symbols.pspeu.stcat.txt

```diff
@@ -68,7 +68,6 @@ EntityHellfireBeast = 0x09255558;
 EntityHellfireBeastThorsHammer = 0x092566B8;
 EntityHellfireBeastFlamePillarAnimation = 0x09257148;
 EntityHellfireBeastFlamePillar = 0x09257850;
-func_us_801CE170 = 0x09257FD0;
```

Comment:
Is there some reason these aren't getting removed by the `make format` step which strips the orphans? Do we need to fix that logic?

Reply:
Weird, I've been running `make format` regularly over the past week and it didn't remove these for me. Maybe it's because I have been doing `build_all` or something /shrug